<launch>
  <node pkg="pose_estimation_odom" name="encoder" type="encoder" output="screen" />
  <node pkg="pose_estimation_odom" name="motor_control_node" type="motor_control_node" output="screen" />
</launch>
